packages/discord.js/src/structures/interfaces/Collector.js

```diff
@@ -97,6 +97,20 @@ class Collector extends EventEmitter {
     if (options.idle) this._idletimeout = setTimeout(() => this.stop('idle'), options.idle).unref();
   }
 
+  /**
+   * The timestamp at which this collector last collected an item
+   * @type {number|null}
```

Comment:
```suggestion
   * @type {?number}
```

Reply:
Some properties on this class are typed as |null others with ? I’m not sure which one to use but |null seems more appropriate

Reply:
Yeah also for me |null is more clear

Reply:
`|null` is used only in 2 places. In all other places `?` is used.

https://github.com/KhafraDev/discord.js/blob/5befbc3773a1cf33614bf900f8301686130b33c7/packages/discord.js/src/structures/interfaces/Collector.js#L83

https://github.com/ckohen/discord.js/blob/8dfd0037b53ec6ef0900a53a0fb1b68fa2cbaf72/packages/discord.js/src/structures/ApplicationCommand.js#L147

Reply:
Exactly, it’s used in this file. I think `?` is for optional properties which this one isn’t

Reply:
<!-- Sorry to burst your bubble --> `?number` and `number|null` are equivalent, the `?` doesn't denote optionality (like in TypeScript), but nullability:

![image](https://user-images.githubusercontent.com/24852502/212279103-69077260-556f-4dbb-8670-0d05747da8f3.png)